knowhere/index/vector_index/ConfAdapterMgr.cpp

```diff
@@ -15,6 +15,12 @@
 
 namespace knowhere {
 
+AdapterMgr&
```

Comment:
these changes are not related with Windows-support, please keep it untouched.

Reply:
Unfortunately, the `implementing singleton in the header has different` behaviors between Linux/GCC and MinGW/GCC.
In MinGW, it will cause create multiple instances, but it does right under Linux (only one instance). I could find some suitable references to explain why so fat, for my understanding, the root cause may be the diff Relocation details between PE and ELF format.